crates/ere-hosts/src/main.rs

```diff
@@ -122,13 +122,13 @@ fn new_risczero_zkvm(
     Ok(EreRisc0::new(program, prover_resource))
 }
 
-fn new_openvm_zkvm(
```

Comment:
You can ignore openVM being commented out. openvm and r0 rely on the same low level c syscall functions so we can't  compile them together